app/src/main/res/xml/preferences.xml

```diff
@@ -76,6 +76,12 @@
     <PreferenceCategory android:title="@string/playback_pref">
         <de.danoeh.antennapod.preferences.SwitchCompatPreference
             android:defaultValue="true"
+            android:enabled="false"
```

Comment:
Ok, this probably really is a bad idea, especially for Android 2.3.x (10) to 4.0 (15)